### ISSUE TITLE
Gamepad config stops on "PREV WPN (P1)"

### ISSUE THREAD

**Issue by MegamixEngine** [MegamixEngine/MegamixEngine#81]
Submitter: Giest118
Email: giest118@gmail.com
I head into the options menu to configure gamepad controls, and after getting to "PREV WPN (P1)", nothing I do causes it to move on. Once this happens, I can only hit Escape to quit the game.

**Comment by RonanDuffy-InkTech** [comment]
I don't have involvement in the engine, but I do believe I found the problem.

In *objOptionsMenu*, in the section of the Step Event called *'Setup Controls'*, I found this switch statement for handling which action to assign a key to.

I'll show where the error occurs:
```
case 'SLIDE':
    global.joystick_slideKey[pID] = key;
    current = 'PREV WPN';
    break;
case 'PREVIOUS WPN':
    global.joystick_weaponSwitchLeftKey[pID] = key;
    current = 'NEXT WPN';
    break;
```
What I believe the problem to be is that the case is **'PREVIOUS WPN'** when it shound be **'PREV WPN'**. This causes a case that the switch statement doesn't acknowledge, leading to a softlock. 

Changing `case 'PREVIOUS WPN':` to `case 'PREV WPN':` seems to fix the issue.
